src/content/getusermedia/record/js/main.js

```diff
@@ -69,11 +69,11 @@ function handleDataAvailable(event) {
 
 function getSupportedMimeTypes() {
   const possibleTypes = [
-    'video/webm;codecs=av1,opus',
     'video/webm;codecs=vp9,opus',
     'video/webm;codecs=vp8,opus',
     'video/webm;codecs=h264,opus',
     'video/mp4;codecs=h264,aac',
+    'video/webm;codecs=av01,opus',
```

Comment:
av01 is a fourcc code as specified in RFC 6381 (ANOTHER namespace for codecs).
Is this enough to be correct, or does it need more parameters?

Reply:
Looks like it is usual only fourcc is specified in MediaRecorder.

Reply:
```suggestion
    'video/webm;codecs=av01,opus',
    'video/webm;codecs=av1,opus', // deprecated, prefer av01
```

Reply:
Done.